### PR TITLE
fix(Bug-6664 : corrected the style on sign out button from warning to …

### PR DESCRIPTION
**Pull request by Sasankachyuta13**
…primary)

### CODE REVIEW

File: assets/i18n/en.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
   "CHECKING_FOR_CLAIMS": "Checking for claims...",
   "DATE_OF_BIRTH": "Date of birth",
   "CREATED_DATE": "Created date",
-  "YOU_ARE_ABOUT_TO_SIGNOUT": "You're about to be signed out",
+  "YOU_ARE_ABOUT_TO_SIGNOUT": "You're about to sign out",
```
Comment (AkhilaHMRC) [inline assets/i18n/en.json:82]: Remove this line, its duplication

**Review by AkhilaHMRC — CHANGES_REQUESTED** [review]
make changes as per comment

**Review by AkhilaHMRC — APPROVED** [review]
make changes as per comment

File: assets/i18n/en.json (hunk 1 above)
Comment (Sasankachyuta13) [reply assets/i18n/en.json]: Yes, removed this akhila.

**Pull request merged**